model_zoo/data_readers/data_reader_imagenet.prototext

```diff
@@ -11,40 +11,24 @@ data_reader {
     percent_of_data_to_use: 1.0
     num_labels: 1000
 
-    image_preprocessor {
-      subtractor {
-        disable: true
-        image_to_sub: "mean-256x256x3-6.bin"
+    transforms {
+      random_resized_aspect_ratio_crop {
+        height: 224
+        width: 224
       }
-
-      cropper {
-        disable: false
-        crop_width: 224
-        crop_height: 224
-        crop_randomly: true
-        resized_width: 256
-        resized_height: 256
-      }
-
-      colorizer {
-        disable: false
-      }
-
-      augmenter {
-        disable: false
-        horizontal_flip: true
-        vertical_flip: false
-        rotation: 0
-        horizontal_shift: 0
-        vertical_shift: 0
-        shear_range: 0
+    }
+    transforms {
+      horizontal_flip {
+        p: 0.5
       }
-
-      normalizer {
-        scale: false
-        subtract_mean: false
-        unit_variance: false
-        z_score: true
+    }
+    transforms {
+      colorize {}
+    }
+    transforms {
+      normalize_to_lbann_layout {
+        means: "0.406 0.456 0.485"
+        stddevs: "0.225 0.224 0.229"
```

Comment:
No, I have no idea what is in that file, since it was never checked in anywhere to my knowledge. These are (one of several possible) standard constants for normalizing ImageNet, and what I have used to reproduce state-of-the-art accuracy.